### PR TITLE
Add dungeonchain

### PR DESCRIPTION
**Pull request by ApeironNodes**

### CODE REVIEW

File: chains/mainnet/dungeonchain.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "chain_name": "Dungeonchain",
+    "api": ["http://dungeonlcd.apeironnodes.com:33001","http://dungeonlcd.apeironnodes.com:33001"],
+    "rpc": ["http://dungeonrpc.apeironnodes.com:33005", "http://dungeonrpc.apeironnodes.com:33005"],
+    "sdk_version": "0.45.5",
```
Comment (Reecepbcups) [inline chains/mainnet/dungeonchain.json:5]: wrong, v0.50

File: chains/mainnet/dungeonchain.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+    "chain_name": "Dungeonchain",
+    "api": ["http://dungeonlcd.apeironnodes.com:33001","http://dungeonlcd.apeironnodes.com:33001"],
+    "rpc": ["http://dungeonrpc.apeironnodes.com:33005", "http://dungeonrpc.apeironnodes.com:33005"],
```
Comment (Reecepbcups) [inline chains/mainnet/dungeonchain.json:4]: Don't seem to work

File: chains/mainnet/dungeonchain.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    "sdk_version": "0.45.5",
+    "coin_type": "4444",
```
Comment (Reecepbcups) [inline chains/mainnet/dungeonchain.json:6]: ? - coin type 118

File: chains/mainnet/dungeonchain.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    "min_tx_fee": "3000",
```
Comment (Reecepbcups) [inline chains/mainnet/dungeonchain.json:7]: fees are 0

**Comment by Reecepbcups** [comment]
@ApeironNodes This is for https://github.com/Crypto-Dungeon/dungeonchain ?

File: chains/mainnet/dungeonchain.json [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    "addr_prefix": "dgn",
```
Comment (Reecepbcups) [inline chains/mainnet/dungeonchain.json:8]: prefix is dungeon

File: chains/mainnet/dungeonchain.json [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+    "logo": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/dungeon/chain.png",
+    "assets": [{
+        "base": "udgn",
+        "symbol": "C4E",
```
Comment (Reecepbcups) [inline chains/mainnet/dungeonchain.json:12]: Symbol: DGN

**Comment by ApeironNodes** [comment]
> @ApeironNodes This is for https://github.com/Crypto-Dungeon/dungeonchain ?

right

**Pull request closed without merging**